packages/pwa-kit-dev/package.json

```diff
@@ -104,7 +104,8 @@
     "webpack-dev-middleware": "^5.2.2",
     "webpack-hot-middleware": "^2.25.1",
     "webpack-hot-server-middleware": "^0.6.1",
-    "webpack-notifier": "^1.12.0"
+    "webpack-notifier": "^1.12.0",
+    "ws": "^8.11.0"
```

Comment:
This version supports a `URL` instance as the `address` argument in the websocket constructor: https://github.com/websockets/ws/blob/master/lib/websocket.js#L666-L677